auth-server/oauth-scim-service/src/main/java/com/google/cloud/healthcare/fdamystudies/oauthscim/controller/LoginController.java

```diff
@@ -156,7 +157,9 @@ public String authenticate(
     String mobilePlatform = cookieHelper.getCookieValue(request, MOBILE_PLATFORM_COOKIE);
     String source = cookieHelper.getCookieValue(request, SOURCE_COOKIE);
     String appName = cookieHelper.getCookieValue(request, APP_NAME_COOKIE);
+    String appVersion = cookieHelper.getCookieValue(request, APP_VERSION_COOKIE);
 
+    auditRequest.setAppVersion(appVersion);
```

Comment:
This should be handled in AuditEventMapper.fromHttpServletRequest(request); for code reusability.

Reply:
Fixed review comment